test/system/035-logs.bats

```diff
@@ -394,6 +394,8 @@ $content--2.*" "logs --until -f on running container works"
 
     # "-t" gives us ^Ms (CRs) in the log
     run_podman run --name $cname --log-driver k8s-file -t $IMAGE echo hi
+    # Sigh.... we can't even trust k8s-file. The log file gets written asynchronously.
```

Comment:
I am still on my first coffee but that doesn't seem right.

The k8s-file gets written by Podman, so on exit of the process the log _must_ be written.  Maybe it's just the comment that confuses me.

Reply:
The file is written by conmon but I agree it should fully be written by the time podman run exits as this should wait for conmon to exit.

Second the test overwrites the file anyway so I do not see how --tail could end up with a partially written log line.

Reply:
My suspicion (unproven) was that conmon was lingering with an open FD. But even if it was, the close() would/should have the file pointer at a place that would not leave an incomplete line.

Anyhow, I just re-pushed with a better approach: `rm` the file before writing to it. That should eliminate the possibility of multiple writers.